docs/en/Configuration-Settings.md

```diff
@@ -45,8 +45,13 @@ Note that, setting Alluxio configuration in this way is application specific and
 
 ## Environment variables
 
-There are a few basic and very frequently used Alluxio configuration properties that can be set via the
-following environment variables:
+> Environment variables are useful to customize configuration of Alluxio when you  
+> [launch Alluxio servers](Running-Alluxio-Locally.html), or [using Alluxio command line interfaces](Command-Line-Interface.html),
+> however, they will not be respected by applications like Spark or MapReduce that use Alluxio as a client. 
```

Comment:
are you asking if setting environment variable works if they are set before a Spark process launched? The answer is no. To make alluxio environment variables work, a process needs to source `libexec/alluxio-config.sh` to translate the   environment variable to Alluxio property opts.

if you are asking whether we need to set environment variable before launching Alluxio processes, the answer is yes.  I updated the doc to make this clear.